openjdk/excludes/ProblemList_openjdk22-openj9.txt

```diff
@@ -433,6 +436,7 @@ java/util/Arrays/TimSortStackSize2.java https://github.com/eclipse-openj9/openj9
 java/util/BitSet/stream/BitSetStreamTest.java https://github.com/eclipse-openj9/openj9/issues/4720 linux-all
 java/util/concurrent/forkjoin/FJExceptionTableLeak.java https://github.com/eclipse-openj9/openj9/issues/3209 generic-all
 java/util/concurrent/locks/Lock/TimedAcquireLeak.java https://github.com/eclipse-openj9/openj9/issues/7125 macosx-all,linux-all,aix-all
+java/util/concurrent/locks/Lock/OOMEInAQS.java https://github.com/eclipse-openj9/openj9/issues/16659 generic-all
```

Comment:
Shouldn't this also be excluded for jdk23?

Reply:
I wasn't tracking a JDK23 exclude file. In that case, I will also add the JDK22 excludes in the JD23 exclude file because the JDK22 failures will most likely also happen in JDK23.

Reply:
Updated the PR to excluded the failing tests for JDK23.